aries-site/src/pages/components/card.mdx

```diff
@@ -98,6 +109,80 @@ TBD
 - Cards that are less promotional should still have enticing titles but do not necessarily need as strong of visual emphasis. The white brand-approved textures are appropriate if a subtle visual emphasis is desired.
```

Comment:
Wondering if everyone knows what "white brand-approved textures" means. Should we direct them on where to find that?

Reply:
Good idea, we can link back to brand central here.